src/libs/events.js

```diff
@@ -45,11 +46,16 @@ export const extractCrossWorkspaceDetails = (fromWorkspace, toWorkspace) => {
 }
 
 export const PageViewReporter = () => {
-  const { name } = useRoute()
+  const { name, params } = useRoute()
 
   useEffect(() => {
-    Ajax().Metrics.captureEvent(`${eventsList.pageView}:${name}`)
-  }, [name])
+    const isWorkspace = /^#workspaces\/.+\/.+/.test(window.location.hash)
+
+    Ajax().Metrics.captureEvent(
+      `${eventsList.pageView}:${name}`,
+      isWorkspace ? extractWorkspaceDetails(params) : undefined
```

Comment:
TOL: `params` isn't really a workspace object. It does happen to have `namespace` and `name` properties, though, so this technically works.

I wonder if we should just always record the URL params that the router pulled out, without any transformation. I think this could be done by adding something like:
```
  appPathParams: Nav.getCurrentRoute().params
```
to the properties here: http://github.com/DataBiosphere/saturn-ui/blob/42443921ba63ce5244a3556e54072ea50946bc14/src/libs/ajax.js#L1347-L1347 (with `Nav.getCurrentRoute()` extracted as a local `const`).

One downside of this is that it won't match how namespace/name are recorded in Mixpanel for events that are using `extractWorkspaceDetails`, but the long-term win of capturing the path parameters might be worth it.

Also, I was a little surprised to find that the `appPath` property we're recording is just the handler's name, not something more like the URL path. Maybe we were worried about privacy? Anyway, I suspect this is where we thought we were recording namespace/name in Mixpanel... but apparently not.